etc/mail.config

```diff
@@ -6,3 +6,7 @@ send_opts=--annotate --cover-letter --no-chain-reply-to --thread
 # initially blocked to avoid mistakenly send more than 15 patches at once to
 # the vger mailing lists. Configure it according to your needs.
 blocked_emails=linux-kernel@vger.kernel.org
+
+# You can add emails to always be included as recipients. Emails should be
+# separated by comma
+additional_emails=
```

Comment:
I don't think this field expresses the idea behind it really well. Maybe we can try something like `demails_always_included` (it is still a bad name, it is just an example)...

We also should have one for `to` and another one for `cc`.